fortran/nf_v2compat.c

```diff
@@ -51,7 +51,9 @@
 #include	"netcdf.h"
 
 #ifndef USE_NETCDF4
+#ifndef NC_CLASSIC_MODEL
```

Comment:
Won't NC_CLASSIC_MODEL always be defined by netcdf.h?

In which case, the definition here should be removed, not placed in an ifdef...